apps/testing/lib/assert_async.ex

```diff
@@ -0,0 +1,83 @@
+defmodule AssertAsync do
+  @moduledoc """
+  Helper macro for making assertions on async actions. This is particularly
```

Comment:
I like this one a LOT, btw

Reply:
Yea this could theorhetically replace eventually.